src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/introduction-new-relic-subscription-usage-data.mdx

```diff
@@ -1872,17 +1872,17 @@ For product-specific attributes and example NRQL queries, keep reading.
     id="synthetics-usage"
     title="Synthetics subscription"
   >
-    For accounts on our [original pricing model](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing), this explains how we calculate billable usage for Synthetics accounts. This document also provides available subscription usage attributes and [example NRQL queries](#query-examples) to use in the New Relic UI or with API calls.
+    For accounts on our [original pricing model](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing), this explains how we calculate billable usage for accounts using synthetic monitoring. This document also provides available subscription usage attributes and [example NRQL queries](#query-examples) to use in the New Relic UI or with API calls.
 
     ### Data generation [#data]
 
-    A New Relic Synthetics subscription level is based on the number of non-[ping](/docs/synthetics-subscription-usage-attributes-queries#label) monitor checks used during a calendar month. Once per day, every Synthetics monitor location will generate an `NrDailyUsage` [event](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types#events-new-relic) for every [monitor type](#type). This event will summarize Synthetics usage for that location and monitor type over the last 24 hours.
+    A synthetics subscription level is based on the number of non-[ping](/docs/synthetics-subscription-usage-attributes-queries#label) monitor checks used during a calendar month. Once per day, every synthetic monitor location will generate an `NrDailyUsage` [event](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types#events-new-relic) for every [monitor type](#type). This event will summarize synthetic monitoring usage for that location and monitor type over the last 24 hours.
```

Comment:
Making this comment here but it would apply to all uses of `synthetics`:

Are we aligned on still using `synthetics` in this case instead of saying `A synthetic monitoring subscription level` instead? Feels slightly weird to call it synthetics in some scenario but not others.